features/projects/components/project-card/project-card.tsx

```diff
@@ -106,6 +105,21 @@ const ViewIssuesAnchor = styled.a`
 
 export function ProjectCard({ project }: ProjectCardProps) {
   const { name, language, numIssues, numEvents24h, status } = project;
+
+  // Used to switch out the default status text returned from API to a preferred text for the UI
+  const modifyStatusTextForUI = (statusText: string) => {
```

Comment:
Exactly what I thought of. One minor note is that you can move this function outside the component which would make it a constant. It's slightly cleaner IMO although it shouldn't make a difference

Reply:
Great suggestion, I wasn't too sure on where to put functions like this. I like the readability improvements when using it outside though.